DataFormats/FTLRecHit/interface/FTLUncalibratedRecHit.h

```diff
@@ -28,6 +28,20 @@ class FTLUncalibratedRecHit {
                         std::pair<float, float> time,
                         float timeError,
                         unsigned char flags = 0);
+  FTLUncalibratedRecHit(const DetId& detId,
+                        std::pair<float, float> ampl,
+                        std::pair<float, float> time,
+                        std::pair<float, float> posit,
+                        float timeError,
+                        unsigned char flags = 0);
+  FTLUncalibratedRecHit(const DetId& detId,
+                        uint8_t row,
+                        uint8_t column,
+                        std::pair<float, float> ampl,
+                        std::pair<float, float> time,
+                        std::pair<float, float> posit,
+                        float timeError,
+                        unsigned char flags = 0);
```

Comment:
Hi,
For the same reason with FTLRecHit, I am not sure whether there are some code using those old constructors so I kept them.
The reason that there is no positionError is that I thought it's important for RecHits to have positionError because we need that to do clustering but we don't need them for UncalibratedRecHit. But now I can add positionError for UncalibratedRecHits and define positionError of RecHits from UncalibratedRecHits, that should be more consistent.

Thanks.